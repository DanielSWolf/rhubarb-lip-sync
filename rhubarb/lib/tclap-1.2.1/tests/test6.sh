#!/bin/sh

# success
../examples/test2 -i 10 -s hello goodbye -ABC > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test6.out; then
	exit 0
else
	exit 1
fi

