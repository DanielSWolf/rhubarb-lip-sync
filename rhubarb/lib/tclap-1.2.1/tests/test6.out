for float we got : 3.7
for int we got : 10
for string we got : hello
for ulabeled we got : goodbye
for bool A we got : 1
for bool B we got : 1
for bool C we got : 1
